src/core/Factory.sol

```diff
@@ -33,7 +34,7 @@ contract Factory is Controllable, ReentrancyGuardUpgradeable, IFactory {
     //region ----- Constants -----
 
     /// @inheritdoc IControllable
-    string public constant VERSION = "1.1.0";
+    string public constant VERSION = "1.1.1";
```

Comment:
such amount of important changes must increase MINOR version, not PATCH. set 1.2.0